source/features/sticky-conversation-sidebar.tsx

```diff
@@ -6,11 +6,20 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import onReplacedElement from '../helpers/on-replaced-element';
 
-const sideBarSelector = '#partial-discussion-sidebar, .discussion-sidebar';
+let sideBarSelector: string;
```

Comment:
No need to hide selectors behind conditions, just use:
```suggestion
const sideBarSelector = [
	'#partial-discussion-sidebar, .discussion-sidebar', // Conversations
	'.repository-content .flex-column > :last-child [data-pjax]' // `isRepoRoot`
];
```